src/mrpro/data/IHeader.py

```diff
@@ -14,18 +17,25 @@
 from mrpro.data.MoveDataMixin import MoveDataMixin
 from mrpro.data.SpatialDimension import SpatialDimension
 from mrpro.utils.summarize_tensorvalues import summarize_tensorvalues
+from mrpro.data.AcqInfo import AcqInfo, mm_to_m, ms_to_s
 
 MISC_TAGS = {'TimeAfterStart': 0x00191016}
+UNKNOWN = 'unknown'
 
 
 @dataclass(slots=True)
 class IHeader(MoveDataMixin):
     """MR image data header."""
 
-    # ToDo: decide which attributes to store in the header
+    b0: float | None
+    """Magnetic field strength [T]."""
+
     fov: SpatialDimension[float]
     """Field of view [m]."""
 
+    h1_freq: float | None
```

Comment:
Can we give a better name?
At least write out frequency?
Proton_lamor_frequency?

Also, is this redundant  to b0?

Reply:
h1_frequency and remove b0